xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -175,8 +176,14 @@ public void requireClientAuth_noClientCert_expectException()
       unaryRpc(/* requestMessage= */ "buddy", blockingStub);
       fail("exception expected");
     } catch (StatusRuntimeException sre) {
-      assertThat(sre).hasCauseThat().isInstanceOf(SSLHandshakeException.class);
-      assertThat(sre).hasCauseThat().hasMessageThat().contains("HANDSHAKE_FAILURE");
+      if (sre.getCause() instanceof SSLHandshakeException) {
+        assertThat(sre).hasCauseThat().isInstanceOf(SSLHandshakeException.class);
```

Comment:
If it enters here because of the `if` clause what's the point of this assertion?

Reply:
To clearly show the assertion being made. So a readability thing. I agree it will never fail, but if the condition is removed/changed in the future, the assertion could easily be forgotten.